using-openspace/ui-panels/navigation/index.md

```diff
@@ -208,14 +206,14 @@ We will also use the Time Panel, which we will cover in more detail.
 
 ### Anchor the Moon, Aim to Earth
 
-[![Moon targeted, aiming to Earth](target_moon_aim_earth.png)](target_moon_aim_earth.png)
+![Moon targeted, aiming to Earth](tutorial_target_moon_aim_earth.png)
 
 {.tutorial}
-1. In the anchor submenu of the panel, confirm that the Moon is set to Anchor.
-2. {octicon}`telescope;1.25em;profile-tour-action` In the Aim submenu, choose Earth. This will keep your spaceship aimed toward Earth and move Earth to the center, behind the Moon.
+1. Open the Anchor and Aim submenu of the panel, confirm that the Moon is set to Anchor.
+2. {octicon}`telescope;1.25em;profile-tour-action` Choose Earth as the Aim by clicking the telescopr button next to Earth. This will keep your spaceship aimed toward Earth and move Earth to the center, behind the Moon.
```

Comment:
```suggestion
2. {octicon}`telescope;1.25em;profile-tour-action` Choose Earth as the Aim by clicking the telescope button next to Earth. This will keep your spaceship aimed toward Earth and move Earth to the center, behind the Moon.
```